recipes/ptychodus/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "ptychodus" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/AdvancedPhotonSource/ptychodus/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: f43b86741f341176133b350474e8511c66e4e26938d3b2592d3c5ec324f4e6f9
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  entry_points:
+    - ptychodus = ptychodus.__main__:main
+
+requirements:
+  host:
+    - pip
+    - python >=3.9
+    - setuptools >=46.4.0
+    - setuptools_scm[toml] >=6.2
```

Comment:
I'm not sure if the brackets annotation works in conda-build. 

In any case you should be fine, since you have setuptools specified separately. 

https://github.com/pypa/setuptools_scm/blob/5ae39d4d7ad9482b242218105d5557462898d547/setup.py#L63

Reply:
```suggestion
    - setuptools_scm >=6.2
```